README.md

```diff
@@ -59,75 +59,146 @@ It is possible to change the behaviour of the data builder by defining environme
-and if tests pass data is copied to 'data/downloads/osm' directory.
+  - `gtfs:dl`
 
-The data is then processed with the following steps:
+  Downloads a GTFS package from a configured location and tests the file with OTP, if
+  the test passes data is copied to the `data/fit/gtfs` directory. The resulting zip file is named `<feedid>.zip`.
 
-3. `gtfs:dl`
-   Downloads a GTFS package from configured location, tests the file with otp, if
-   test passes data is copied to directory 'data/fit/gtfs'. The resulting zip is named <feedid>.zip.
+  - `gtfs:fit`
 
-4. `gtfs:fit`
-   Runs configured map fits. Copies data to directory 'data/filter/gtfs'.
+  Runs configured map fits. Copies data to the `data/filter/gtfs` directory.
 
-5. `gtfs:filter`
-   Runs configured filterings. Copies data to directory 'data/id/gtfs'.
+  - `gtfs:filter`
 
-6. `gtfs:id`
-   Sets the gtfs feed id to <id> and copies data to directory 'data/ready/gtfs'.
+  Runs configured filters. Copies data to the `data/id/gtfs` directory.
 
-Steps 3. - 6. can also be run together using a single `gtfs:update` command.
+  - `gtfs:id`
 
-Building the router from available (seeded or downloaded and processed) data:
+  Sets the gtfs feed id to `<id>` and copies data to the `data/ready/gtfs` directory.
 
-7. `router:buildGraph`
+- `router:buildGraph`
+  - `router:copy`
+  - `buildOTPGraphTask(config.router)`
 
 Builds a new graph with all the new data sets (and maybe seeded data sets if there were issues with new data).
 
-8. `test.sh`
+- `router:buildStreetOnlyGraph`
+  - `router:copyStreetOnlyGraphData`
+  - `buildOTPStreetOnlyGraphTask(config.router)`
 
-Runs routing quality test bench defined in the repository 'hsldevcom/OTPQA'. OTPQA test sets are associated with GTFS packages.
-If there are quality regressions, a comma separated list of failed GTFS feed identifiers is is written to local file 'failed_feeds.txt'.
+Builds a new street only graph with all the new data sets (and maybe seeded data sets if there were issues with new data).
 
-9. `router:store`
+- `router:buildWithPrebuiltStreetGraph`
+  - `router:copyForPrebuiltStreetGraphDataBuild`
+  - `buildOTPGraphTask(config.router)`
+
+Builds a new graph from prebuilt street only data with new gtfs data sets (and maybe seeded data sets if there were issues with new data).
+
+- `test.sh`
+
+Runs routing quality test bench defined in the repository `hsldevcom/OTPQA`. OTPQA test sets are associated with GTFS packages.
+If there are quality regressions, a comma separated list of failed GTFS feed identifiers is is written to local file `failed_feeds.txt`.
+
+- `router:store`
 
 Stores the new data in storage (which can be a mounted storage volume).
 
-10. `deploy.sh`
+- `router:storeForPrebuiltStreetGraphDataBuild`
+
+Stores the new data in storage (which can be a mounted storage volume). Also copies the `report` directory from the street only build to the output directory under the name `osm-report`.
```

Comment:
`osm-report` is a bit misleading since some of the problems are related to DEM data etc. So maybe `street-report`?

Reply:
I changed the name to street-report